### PR TITLE
Add design principles

### PR DESCRIPTION
**Pull request by jtamiace**

### CODE REVIEW

File: docs/pages/principles.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<template>
+
+  <DocsPageTemplate>
+
+    <DocsPageSection title="" anchor="">
```
Comment (indirectlylit) [inline docs/pages/principles.vue:5]: Could this have simply been 

```html
<DocsPageSection>
```

Alternatively, was the section necessary at all?

Comment (jtamiace) [reply docs/pages/principles.vue:5]: I was getting linting errors without adding that part in

Comment (indirectlylit) [reply docs/pages/principles.vue:5]: okay thanks - opened a follow-up issue: https://github.com/learningequality/kolibri-design-system/issues/96

Comment (indirectlylit) [reply docs/pages/principles.vue:5]: FYI, this is addressed in https://github.com/learningequality/kolibri-design-system/pull/114

You can now have a `DocsPageSection` with no title and anchor

**Pull request merged**